core/load-git.c

```diff
@@ -246,6 +253,9 @@ static void parse_dive_airtemp(char *line, struct membuffer *str, void *_dive)
 static void parse_dive_watertemp(char *line, struct membuffer *str, void *_dive)
 { UNUSED(str); struct dive *dive = _dive; dive->watertemp = get_temperature(line); }
 
+static void parse_dive_airpressure(char *line, struct membuffer *str, void *_dive)
+{ UNUSED(str); struct dive *dive = _dive; dive->surface_pressure = get_airpressure(line); }
```

Comment:
wouldn't we have to set that `userflag` or better `surface_pressure_calculated` flag to `false` here?
I guess we could rely on it being zero initialized (I'd have to spend more time thinking through the code to make sure that this is indeed the case)